package.json

```diff
@@ -12,16 +12,16 @@
     "build": "node build.js && chmod +x bin/setup.sh"
   },
   "dependencies": {
-    "@typescript-eslint/parser": "^5.17.0",
-    "eslint-config-prettier": "^8.5.0"
+    "@typescript-eslint/parser": "^6.8.0",
+    "eslint-config-prettier": "^9.0.0"
   },
   "devDependencies": {
-    "@trivago/prettier-plugin-sort-imports": "^3.2.0",
-    "@typescript-eslint/eslint-plugin": "^5.17.0",
-    "@typescript-eslint/parser": "^5.17.0",
-    "eslint": "^8.19.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "prettier": "^2.6.1"
+    "@trivago/prettier-plugin-sort-imports": "^4.2.0",
```

Comment:
@simonpratt what are these dev dependencies actually used for? Are they needed?

Reply:
I not really sure. From memory the packages required to use prettier with eslint have change quite a bit since I last touched this package.json

Reply:
I think most of the packages are probably needed by the **consumer**. Which your nice cli utility installs on their behalf 😄 

I'm just wondering what the dev dependencies are for, maybe they're here with a view to implementing some tests of the linting setup in this repo in the future?